<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DotEditor</class>
 <widget class="QMainWindow" name="DotEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>409</width>
    <height>322</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DotEditor</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>381</width>
      <height>211</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>361</width>
      <height>27</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>format</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_format">
       <property name="editable">
        <bool>false</bool>
       </property>
       <item>
        <property name="text">
         <string>jpg</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>png</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>svg</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>engine</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_engine">
       <item>
        <property name="text">
         <string>dot</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_show">
       <property name="text">
        <string>show</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>409</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionHelp"/>
   </widget>
   <addaction name="menuEdit"/>
  </widget>
  <action name="actionAbout">
   <property name="text">
    <string>about</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>help</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
